timescaledb/tutorials/custom-timescaledb-dashboards.md

```diff
@@ -0,0 +1,305 @@
+* How to build your React front-end to visualize the data
+
+The project uses React, connecting to a [Hasura][] GraphQL API to visualize [hypertable chunks][hypertables] of a [TimescaleDB][] instance. 
+
+The easiest way to get a TimescaleDB instance is to [try for free][timescale-signup] using our hosted service. You can also [download TimescaleDB for free][timescale-install] and run locally or in your own cloud infrastructure.
+
+You can get the full code for this project from [this GitHub repo][repo-example].
+
+This project works on any TimescaleDB instance, but if you’re interested in generating sample data to use, use our [Simulating IoT sensor data][iot-tutorial] tutorial.
+
+## How TimescaleDB manages time-series data
+
+TimescaleDB uses [hypertables][] to store time-series data. TimescaleDB automatically partitions data in hypertables into smaller child tables called chunks. The chunks represent data for a given time period, which makes it easier to query and manage over time. For example, if you wanted to query data from 10am to 11am, instead of scanning your entire database, TimescaleDB would scan the specific chunks that contain data for just that period. All the interaction with the database still occurs on the hypertable using SQL, but TimescaleDB partitions the data to make large queries more efficient. 
+
+Many features in TimescaleDB rely on chunks, including [continuous aggregates][caggs], [data retention][], and  native [compression][]. Native compression is particularly helpful with large time-series datasets. Time-series data can be relentless in quantity and speed, and difficult to store and query without a purpose-built time-series database. You can use TimescaleDB compression to save as much as 97% of your disk space for the same amount of data, and usually increase the speed of your queries over time. 
+
+Visualizing the state of your hypertables can help you gain a better understanding of how  compression works and possibly even how different types impact compression efficiently. Visualization can help you see the results of compression table by table, and chunk by chunk. To do this, TimescaleDB provides multiple views and functions that can be queried for information about the state of your hypertables and chunks. Although there is no combined view that provides exactly the data we need for our visualization, TimescaleDB provides the building blocks to craft a custom SQL query that returns the data needed to better visualize the current hypertable and chunk compression state. For example, this query returns the name and time series range that this chunk covers:
+
+```sql
+tsdb=> SELECT chunk_name, range_start, range_end FROM timescaledb_information.chunks LIMIT 1;
+    chunk_name    |      range_start       |       range_end
+------------------+------------------------+------------------------
+ _hyper_2_2_chunk | 2021-04-29 00:00:00+00 | 2021-05-06 00:00:00+00
+(1 row)
```

Comment:
GH formatting is screwing up when I try to suggest this, but you need to close this code block.